kolibri/core/content/test/test_zipcontent.py

```diff
@@ -301,6 +308,11 @@ def test_delete_not_allowed(self):
         self.assertEqual(response.status_code, 405)
 
 
+@override_option("Deployment", "ZIP_CONTENT_URL_PATH_PREFIX", "prefix_test/")
+class UrlPrefixZipContentTestCase(ZipContentTestCase):
+    pass
```

Comment:
Is this test just testing that the option exists?

Reply:
No, it's rerunning the whole test case with the option set.